src/components/Text/types.ts

```diff
@@ -4,4 +4,5 @@ export interface TextProps extends SpaceProps {
   color?: string;
   fontSize?: string;
   bold?: boolean;
+  textTransform?: string;
```

Comment:
you can narrow down the values to `"uppercase" | "lowercase" | "capitalize"`, or maybe use a boolean `uppercase` ?

Reply:
Aye great shout, sorry I should have had that in from the start. Pro the strings > an `uppercase` boolean